### ISSUE TITLE
[stable/*] bump dependency mariadb 1.0.3

### ISSUE THREAD

**Issue by scottrigby** [helm/charts#1944]
## PRs to review
- [ ] drupal #TBD 
- [ ] ghost #TBD 
- [ ] jasterreports #TBD 
- [ ] jooma #TBD 
- [ ] magento #TBD 
- [ ] mediawiki #TBD     
- [ ] moodle #TBD 
- [ ] opencart #TBD 
- [ ] orangehrm #TBD  
- [ ] osclass #TBD 
- [ ] owncloud #TBD 
- [ ] phabricator #TBD 
- [ ] phpbb #TBD 
- [ ] prestashop #TBD 
- [ ] redmine #TBD 
- [ ] sugarcrm #TBD 
- [ ] suitecrm #TBD 
- [ ] testlink #TBD 
- [ ] wordpress #TBD 

## Background
PRs opened using my dependency version bump utility script. See:
- https://github.com/kubernetes/helm/issues/1947
- https://github.com/kubernetes/helm/pull/2041
- https://github.com/kubernetes/helm/issues/2205

**Comment by scottrigby** [comment]
This one is on hold until #2004 is merged:

From  @prydonius on [Slack](https://kubernetes.slack.com/archives/C6E3XH1ED/p1504716789000248?thread_ts=1504551308.000078&cid=C6E3XH1ED):
> there is a PR for updating the storageClass pattern in MariaDB but it hasn't been merged yet. We should merge that first, and then update the other PRs for storageClassName to pull in the new mariadb before merging

**Comment by scottrigby** [comment]
This issue will also be updated to MariaDB `v1.0.3`:

From @mgoodness on [Slack](https://kubernetes.slack.com/archives/C6E3XH1ED/p1504551308000078):
> @scottrigby I merged MariaDB to v1.0.3 this morning. how do you want to handle the v1.0.2 dependency bump PRs?

Tl;dr, I'm going to close the 1.0.2 PRs, and reopen 0.1.3 equivalents, and edit this issue to point to those new PRs.
